### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by shagunbagrecha**
Cart Page

Products added to cart.

Categorization:
- Name 
- Price
- subtotal

Preview:
https://deploy-preview-3--hitfit.netlify.app/

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import '/css/nav.css';
+
+:root{
```
Comment (AbhishekkGautam) [inline css/cart.css:3]: we can create a default/common CSS file for root variable names & import it here. later on, it'll easier to change the global styles in just one file.

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	--secondary-color:#393939;
+}
+
+.cart-page{
+	margin: 80px auto;
+}
+
+table{
```
Comment (AbhishekkGautam) [inline css/cart.css:11]: we can avoid giving styles directly to the element.

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+	 <title>Cart | HitFit</title>
+</head>
+<body>
+	 <!-- navbar -->
+	 <nav class="navbar">
+		<div class="nav  d-flex flex-justify-between flex-items-center p-1">
+			<h1 class="brandname h3">HitFit</h1>
+			<div class="nav-items d-flex flex-items-center">
+				<div class="search d-flex">
+					<input type="text" class="search-box" placeholder="Search..">
```
Comment (AbhishekkGautam) [inline pages/cart.html]: as an alternative, we can also give type="search" for search-related input. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search

**Review by AbhishekkGautam — APPROVED** [review]
Everything else looks good to me. clean code. ✅

**Review by gopalbbharadva — APPROVED** [review]
nice work!!

File: css/cart.css (hunk 1 above)
Comment (shagunbagrecha) [reply css/cart.css:3]: Understood! Thanks for the review, I'll refactor the code once I'm done with the submission.

**Pull request merged**